third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp

```diff
@@ -362,42 +362,49 @@ bool init(const char** libs, int length)
     // spec, so wrappers may be needed if they are not exported by the OpenCL
     // runtime library.
     MAP_FUNC(clReleaseDevice)
-    if (!clReleaseDevice)
-        MAP_FUNC_TO_WRAPPER(clReleaseDevice, 1, 1)
+    if (!clReleaseDevice) {
```

Comment:
I just realized why you have to add the curly braces. It's actually a defect in the macro definition. Please try the do {} while (0) trick instead.